gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.1-bin.zip
```

Comment:
We tried upgrading gradle, but any version above 5 breaks the maven publish plugin for some reason. Is this necessary?

Reply:
spring boot 2.6 requires gradle 6.8 or higher :(

"Spring Boot’s Gradle plugin requires Gradle 6.8, 6.9, or 7.x"